### PR TITLE
modulesync 6.0.0

### PR DESCRIPTION
**Pull request by bastelfreak**
modulesync 5.3.0

### CODE REVIEW

**Comment by canihavethisone** [comment]
the `add_mocked_facts!` line 12 of spec_helper.rb is breaking unit tests currently

**Review by smortex — APPROVED** [review]
LGTM!

I worked with @canihavethisone on the CI fix so will wait for someone else review before merging!

**Pull request merged**
